.github/workflows/verify-build.yml

```diff
@@ -15,6 +15,9 @@ env:
   PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
   GO_VER: 1.18.2
 
+permissions:
```

Comment:
don't we miss the
```
permissions: {}
```
directive here?

Reply:
{} means none. It is is also possible, but then we need to add read in every single workflow that does check out.